packages/@lwc/template-compiler/src/__tests__/fixtures/parsing-errors/invalid-template-attribute/multiple-attributes-mixed/actual.html

```diff
@@ -1,13 +1,6 @@
 <template>
     <section>
         <template if:true={showItems} for:each={items} for:item="item" onfoo={handleFoo}>
-            <!-- 
```

Comment:
This scenario will now throw an error instead of ignoring the invalid attribute (`onfoo`)